### ISSUE TITLE
Create a home page design

### ISSUE THREAD

**Issue by mhdeyad** [mhdeyad/myappsample#1]
this is for create home page design

**Comment by mbnoimi** [comment]
This is incorrect because the **Issues** section is just for bug reports or support requests not for todos :)
